src/components/CreateNote/CreateNoteContainer.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  UseQueryResult,
+  useMutation,
+  useQuery,
+  useQueryClient,
+} from '@tanstack/react-query';
+import { v4 as uuid } from 'uuid';
+
+import { fetchNotesOfUser, notesActions } from 'api';
+import { INote } from 'types';
+import { useFieldValue, useToggle } from 'hooks';
+
+import CreateNote from './CreateNote';
+import { setNewId } from 'utils';
+import { useAppSelector } from 'store/hooks';
+import { getUser } from 'store/selectors';
+
+const CreateNoteContainer = () => {
+  const queryClient = useQueryClient();
+  const { user } = useAppSelector(getUser);
+  const { data: notes }: UseQueryResult<INote[], Error> = useQuery<
```

Comment:
it's better to rename data like `notesResponse` to show that we receive data from BE